resources/views/layouts/_horizontal/menu_container.blade.php

```diff
@@ -4,7 +4,7 @@
             <div class="{{ backpack_theme_config('options.useFluidContainers') ? 'container-fluid' : 'container-xl' }}">
                 <ul class="navbar-nav">
                     @unless(backpack_theme_config('options.doubleTopBarInHorizontalLayouts'))
-                        <li>
+                        <li class="nav-brand">
```

Comment:
This class needs to be applied to the other horizontal styles too:
- horizontal_overlap
- horizontal_dark

Because I notice they have the same problem:
<img width="1512" alt="CleanShot 2023-06-27 at 12 15 00@2x" src="https://github.com/Laravel-Backpack/theme-tabler/assets/1032474/1fdfcef0-abcf-42c7-abeb-0e36d9b8abe2">
<img width="1512" alt="CleanShot 2023-06-27 at 12 15 25@2x" src="https://github.com/Laravel-Backpack/theme-tabler/assets/1032474/a5006635-aa22-4a90-a5bd-ebeb22c505f0">